playbook.yml

```diff
@@ -41,6 +42,16 @@
      when: provision_postgres == 'true'
      become: true
 
+- name: Provision prometheus
+  become: true
+  vars:
+   provision_prometheus: "true"
+  hosts: prometheus-hosts
+  roles:
+   - role: prometheus
+     when: provision_prometheus == 'true'
```

Comment:
we should document this `provision_prometheus` variable in the README file, together with the other `provision_` variables.
At the end, as prometheus is coming from non supported repos, we need to tell the user that they can even use their own prometheus installation